client/executor/wasmtime/src/instance_wrapper.rs

```diff
@@ -72,6 +73,69 @@ impl ModuleWrapper {
 	}
 }
 
+/// Format of the entrypoint.
+pub enum EntryPointType {
+	/// Direct call.
+	Direct,
+	/// Indirect call.
```

Comment:
so that means that the entrypoint is the wrapper and the inner u32 is the index in the table? Can we leave a doc about this?